docs/third-party-developers/extensibility/hooks/actions.md

```diff
@@ -39,13 +40,15 @@
  - [woocommerce_store_api_rate_limit_exceeded](#woocommerce_store_api_rate_limit_exceeded)
  - [woocommerce_store_api_validate_add_to_cart](#woocommerce_store_api_validate_add_to_cart)
  - [woocommerce_store_api_validate_cart_item](#woocommerce_store_api_validate_cart_item)
+ - [woocommerce_{$product->get_type()}_add_to_cart](#woocommerce_-product--get_type-_add_to_cart)
+ - [{$hook}](#-hook)
```

Comment:
Is it correct that we show in the documentation this?

Reply:
This is auto-generated by the `npm run build:docs` command, so unless we change the way this script is wired, there's not much we can do

Reply:
There is a way to ignore the hooks https://github.com/wp-hooks/generator#ignoring-hooks

Do you think we should try? What do you think?

Reply:
Well, since this discussion started after the PR was already merged, I'd recommend opening a separate issue and working on it in the next cooldown.

Reply:
I created an issue https://github.com/woocommerce/woocommerce-blocks/issues/8540.